src/screens/LeaderboardScreen.js

```diff
@@ -13,6 +18,27 @@ export class LeaderboardScreen extends Component<Props> {
     return <Text>Hi</Text>;
   };
 
+  _renderLeaderBoard = (item, index) => {
+    return (
+      <View style={styles.leaderboard}>
+        <View style={styles.leaderboard__deckName}>
+          <TouchableOpacity
+            onPress={() =>
+              alert(`W: ${item.wins} L: ${item.games - item.wins}`)
+            }
+          >
+            <Text>
+              {index + 1}. {item.commander}
+            </Text>
+          </TouchableOpacity>
```

Comment:
Is this that  `.` a typo? Also, personal opinion, but I like to keep smaller objects on a single line. 
`<Text>{index + 1}. {item.commander}</Text>`